### DISCUSSION TITLE
Basic question: how to automatically change sources?

### DISCUSSION THREAD

**Discussion by rice1204** [awawa-dev/HyperHDR#990] [Q&A] [answered]
Hi HyperHDR community. I've just started out with HyperHDR and have gotten the basic setup running well. Thanks @awawa-dev for the amazing work!

Pardon the basic question: How do you automate changing sources?

I have 2 devices, an nvidia shield and a PS5. The Shield is grabbed with HyperionGrabber (priority 160) and the PS5 is grabbed via USB routed through ezcoo splitter (priority 180). 

I would like to automate the process of changing sources. I assumed it would work as when the PS5 is turned on, the higher priority assignment would automatically take over, and when the PS5 turned off, it would revert to the lower priority shield. However, this is not the case (at least I haven't been able to configure it to work). The only way I've found to change sources so far is by manually going to **Remote Control -> click source Active**

Alternatively, I could use Home Assistant to send json triggers, but I don't see any functionality to set the source in the json playground or docs.

Is there something obvious I have overlooked?  

Thanks!

**Comment by awawa-dev** [comment, marked as answer]
Hi
Did you test HyperHDR 21 alpha for ex. https://github.com/awawa-dev/HyperHDR/actions/runs/11875476246 ? Full logs (uploaded on pastebin or something similar) recording the few minutes in which both sources are registered and then the one with the lower priority becomes inactive may be helpful.

**Reply by rice1204** [reply]
Thanks! I was not aware 21 alpha was available. I will try to compile it and report back.

**Reply by awawa-dev** [reply]
You can use already built installers from that link.

**Reply by rice1204** [reply]
Sorry, please disregard! It was my own user error. 
Turns out I had the priorities set backwards. Everything working as expected.
